dds_cli/account_manager.py

```diff
@@ -274,3 +275,37 @@ def find_user(self, user_to_find: str) -> None:
         LOG.info(
             "Account exists: [bold]%s[/bold]", "[blue]Yes[/blue]" if exists else "[red]No[/red]"
         )
+
+    def save_emails(self, unit: str = None) -> None:
+        """Get user emails and save them to a text file."""
+        # Get emails from API
+        response, _ = dds_cli.utils.perform_request(
+            endpoint=dds_cli.DDSEndpoint.USER_EMAILS,
+            method="get",
+            headers=self.token,
+            json={"unit": unit, "all": unit is None},
```

Comment:
It seems this `json` part of the request is not used for anything at the moment. I guess the idea is that one should be able to use the command for a specific unit only. Since this can be confusing, I think we should either have the implementation for this in the endpoint, or we should remove `json` from the command.

Reply:
Agreed. This was for when I was implementing the more complicated version of the API Endpoint. Will remove for now and we can look at this later.